web-ui/src/main/resources/catalog/views/default/less/gn_search_default.less

```diff
@@ -447,4 +446,60 @@ html, body {
 .navbar-form {
   border:none;
 }
+
+.gn-icon-bg:hover {
+  color: #fff;
+  background-color: #428BCA;
+  text-decoration:none;
+}
+
+.gn-icon-bg .badge {
+  background-color: #428BCA;
+}
+  
+.gn-icon-bg:hover .badge {
+  background-color: #ffffff;
+  color: #428BCA;
+}
+
+/* used to display icons with resource types - iso19139 hierarchylevel */
+.gn-icon-dataset:before { content: @fa-var-database; }
+.gn-icon-service:before { content: @fa-var-cog }
+.gn-icon-map:before { content: @fa-var-globe }
+.gn-icon-staticMap:before { content: @fa-var-globe }
+.gn-icon-featureCatalog:before { content: @fa-var-tag}
+.gn-icon-document:before { content: @fa-var-file } 
+.gn-icon-attribute:before { content: @fa-var-tag }      
+.gn-icon-attributeType:before { content: @fa-var-sliders }       
+.gn-icon-collectionHardware:before { content: @fa-var-video-camera }       
+.gn-icon-collectionSession:before { content: @fa-var-truck }            
+.gn-icon-series:before { content: @fa-var-files-o }       
+.gn-icon-nonGeographicDataset:before { content: @fa-var-bar-chart-o }       
+.gn-icon-dimensionGroup:before { content: @fa-var-star-half-o }       
+.gn-icon-feature:before { content: @fa-var-tag }       
+.gn-icon-featureType:before { content: @fa-var-sitemap }       
+.gn-icon-propertyType:before { content: @fa-var-sliders }       
+.gn-icon-fieldSession:before { content: @fa-var-calendar } 
+.gn-icon-software:before { content: @fa-var-floppy-o }             
+.gn-icon-model:before { content: @fa-var-dashboard }
+/* GN-categories */ 
```

Comment:
Sounds good to use css icon for categories, but that means also to:
* update a number of other locations like https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/metadataactions/partials/metadatacategoryupdater.html, https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/templates/admin/classification/categories.html, https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/category/partials/category.html
* drop the existing image from the image/category folder.